### ISSUE TITLE
Able to run different testcases under a folder

### ISSUE THREAD

**Issue by muneebullashariff** [mbits-mirafra/i3c_avip#29]
- Instead of using svunit.f for each unit_test, can we include all these inside the unit_test file itself - but htis we need to see how to keep the path for the source code intact and not messy
- Continute to use svunit.f files and package files, but find a way to automatically grep the svunit.f files 

`runSVUnit -f controller_agent_config_unit_test/svunit.f -f controller_tx_unit_test/svunit.f `
Solve the relative path problem using ENV variables 

Basically just by using the below command, I should be able to run all the testcases from the current folder and sub-folders. 
`runSVUnit`

**Comment by muneebullashariff** [comment]
Also write project.csh file under the repo with different ENV variables 
SRC
SIM
UNIT_TESTS
PROJECT

**Comment by muneebullashariff** [comment]
Question: How to include multiple directories and sub-directories using +incdir+
+incdir+ works ONLY for `inlcude "files.sv" - basically, when the file is mentioned inside the `include "" then the compiler will check for those files names inside the +incdir+ folders mentioend.

-f file_list.f can have +incdir+ and files that needs to get compiled but can't have `include.

**Comment by muneebullashariff** [comment]
![Image](https://github.com/mbits-mirafra/i3c_avip/assets/15922511/85a74beb-72fa-42a8-b428-a202ff2892c1)

include then import order
